doc/content/getting-started/migrating-from-v2/import-devices.md

```diff
@@ -14,6 +14,7 @@ $ ttn-lw-cli end-devices create --application-id "imported-application" < device
 This will import your devices on {{% tts %}}. In case any device fails, you see a relevant error message at the end of the output.
 
 If the import was successful, your devices is added to the list of end-devices in your application.
-{{< figure src="../migrating-from-networks/successful-import.png" alt="successful-import" >}}
+
+{{< figure src="../successful-import.png" alt="successful-import" >}}
```

Comment:
I don't think this is right, the figure is in the `migrating-from-networks` folder. Did you run locally and check?

Reply:
Yes, tested it locally.